icon_validator/rules/workflow_validators/workflow_help_plugin_utilization_validator.py

```diff
@@ -0,0 +1,92 @@
+from icon_validator.rules.validator import KomandPluginValidator
+from icon_validator.exceptions import ValidationException
+
+import os
+import json
+import re
+
+
+class WorkflowHelpPluginUtilizationValidator(KomandPluginValidator):
+
+    @staticmethod
+    def extract_plugins_used(spec: object) -> list:
+        value = spec.directory
+        data = dict()
+        for file_name in os.listdir(value):
+            if file_name.endswith(".icon"):
+                with open(f"{value}/{file_name}") as json_file:
+                    try:
+                        data = json.load(json_file)
+                    except json.JSONDecodeError:
+                        raise ValidationException(
+                            "The .icon file is not in JSON format. Try exporting the .icon file again")
+        try:
+            plugins = data["kom"]["workflowVersions"][0]
+        except KeyError:
+            raise ValidationException("The .icon file is not formatted correctly. Try exporting the .icon file again")
+
+        plugin_list = list()
+        try:
+            for step in plugins["steps"]:
+                if "plugin" in plugins["steps"][step].keys():
+                    # The plugin_list must have a len of 1 or more for the rest of the code to work,
+                    # so the first item is just added
+                    if len(plugin_list):
+                        for index, d in enumerate(plugin_list):
```

Comment:
I *think* this can be reduced...but I'm not sure. 

```
if len(plugins)
   add 1 if found
   append if not found
if not found
   append
```

to 

```
if len(plugins)
   add 1
append
```

Maybe...I'm likely wrong here though and want a 2nd opinion.